strawberryfields/ops.py

```diff
@@ -1361,6 +1362,30 @@ def _apply(self, reg, backend, **kwargs):
         backend.beamsplitter(t.x, r.x, *reg)
 
 
+class MZgate(Gate):
+    r"""Mach-Zehnder interferometer.
+
+    .. math::
+        MZ(\phi_1, \phi_2) = BS(\pi/4, \pi/2) (R(\phi_1)\otimes I) BS(\pi/4, \pi/2) (R(\phi_2)\otimes I)
+
+    Args:
+        phi1 (float): internal phase
+        phi2 (float): external phase
```

Comment:
Maybe consider swapping the names phi1 and phi2? Temporally phi2 takes place first.

Reply:
I did this in order to match the order provided by the `rectangular_symmetric` decomposition - if it makes more sense, we should change both.

Reply:
Done